README.md

```diff
@@ -5,10 +5,20 @@
 ```bash
 docker build -t urc_software .
 export DISPLAY=128.180.198.214:0.0 # replace with your IP address up to the colon
-docker run -it -e DISPLAY=$DISPLAY --net=host urc_software
+```
+
+### Tell docker to clean up after itself when it exits
+
+#### In powershell or command prompt
+
+```bash
+docker run --rm -v C:/Users/phamd/urc_software:/root --name urc_software_dev -it -e DISPLAY=$DISPLAY urc_software

```

Comment:
Why do we need to mourn a volume with the source if the source is already in the docker image?

Reply:
So I was mounting so I can edit code outside of the container and have those changes automatically available inside the running container without needing to rebuild the Docker image each time. We won't need to do this on the rover since the container will not be modified during execution.

Reply:
ok I see so it's just a temporary fix for debugging. But that will only work with scripts and python not C++ right? because c++ still needs to be compiled